layouts/pricing/index.html

```diff
@@ -0,0 +1,500 @@
+             </p> -->
+             <p>
+               <a href="mailto:cloud-gov-inquiries@gsa.gov?subject=Purchasing%20FISMA%20Package" class="button">Contact us</a><br/>
+               <a href="/overview/pricing/rates/#fisma-low" class="small-link">About FISMA packages &rarr;</a>
+             </p>
+           </li>
+         </ul>
+      </section>
+
+      <section class="usa-grid">
+        <h2>All use of cloud.gov includes:</h2>
+        <!-- <p>
+          Placeholder text.
+        </p> -->
+        <div class="usa-width-one-half">
+          <ul class="column-single">
+            <li>
+              FedRAMP JAB-authorized hosting
+            </li>
+            <li>
+              Self-service provisioning of services
+            </li>
+            <li>
+              Self-service deployment of apps
+            </li>
+            <li>
+              Basic technical and compliance support (during business hours)
+            </li>
+            <li>
+              Documentation/tutorial material
+            </li>
+            <li>
+              Continuous platform-level scanning and security updates
+            </li>
+          </ul>
+
+
+            <p>
+              Free services
+            </p>
+
+            <ul class="column-single">
+              <li>
+                Postgres database
+              </li>
+              <li>
+                MySQL database
+              </li>
+              <li>
+                Elasticsearch RESTful search and analytics
+              </li>
+              <li>
+                Redis in-memory data store
+              </li>
+              <li>
+                S3 object store
+              </li>
+            </ul>
+
+
+        <!-- <div class="usa-width-one-half">
+          <p>
+            Access to free services
+          </p>
+
+          <div>
+            <ul class="column-single">
+              <li>
+                Postgres database
+              </li>
+              <li>
+                MySQL database
+              </li>
+              <li>
+                Elasticsearch RESTful search and analytics
+              </li>
+              <li>
+                Redis in-memory data store
+              </li>
+              <li>
+                S3 object store
+              </li>
+            </ul>
+          </div> -->
+
+        </div>
+      </section>
+
+      <section class="usa-grid">
+
+        <h2>Paid packages also include:</h2>
+
+        <div class="usa-width-one-half">
+          <ul class="column-single">
+              <li>
+                Self-service management of spaces and user access
+              </li>
+              <li>
+                Single-sign-on with your SAML identity provider
+              </li>
+              <li>
+                Access to premium add-on services
+              </li>
+          </ul>
+
+          <p>
+            Premium services &mdash; free for a limited time
```

Comment:
I think "free for a limited time" doesn't provide any clarity over just saying "Premium services include"

It's not as if Netflix is priced at "$8.99 for a limited time" - it's implicit in the price of anything that future pricing will be different. And it's not clear if "free for a limited time" means you get the first six months, say, of your services for free, then start paying.

This is a minor point, however, and I rather ship this and bandy about the wording later if we need to.

Reply:
If we don't telegraph that this is coming, It's not just that pricing will go up, it's that the model will change. We want them to "get" the model and not have to readjust that understanding if possible so that it will be non-controversial when it starts appearing in their renewals and bills.